src/app/shared/services/plot.service.ts

```diff
@@ -516,13 +516,13 @@ export class PlotHelperService {
 
         _chart.yAxisMin = function() {
             const padding = _yRangePadding * yAxisRangeRatio();
-            const min = parseInt(d3.min(_chart.data(), function(g) { return g['min']; }), 10) - 1;
+            const min = parseInt(d3.min(_chart.data(), function(g) { return g['axis_min'] || g['min']; }), 10) - 1;
```

Comment:
Is gene.logfc not always defined?  When is g['axis_min'] or g['axis_max'] undefined, requiring this fallback value?

Reply:
This file contains an "extended/tweaked" version of D3 Boxplot which can be used to generate any type of box plot. In this case, the plot has an additional layer with dots (gene.logfc) generated independently from the boxes min/max. Using "axis_min" and "axis_max" gives options to override the y scale only if needed. Does that make sense?